packages/documentation/src/stories/foundation/layout/grid/grid.blocks.tsx

```diff
@@ -1,122 +1,60 @@
-              {data.value.infix === 'none' ? 'none' : <code>{`-${data.key}-`}</code>}
-            </td>
-          ))}
-        </tr>
-
-        <tr>
-          <th>
-            Container <code>max-width</code>
-          </th>
-          {forEach(
-            SCSS_VARIABLES.container,
-            (data: { key: string; value: { ['max-width']: string } }) => (
-              <td key={data.key}>{data.value['max-width']}</td>
-            ),
-          )}
-        </tr>
-        <tr>
-          <th>
-            Container <code>padding</code>
-          </th>
-          {forEach(
-            SCSS_VARIABLES.container,
-            (data: { key: string; value: { padding: string } }) => (
-              <td key={data.key}>{data.value.padding}</td>
-            ),
-          )}
-        </tr>
-
-        <tr>
-          <th>
-            Container-Fluid <code>padding</code>
-          </th>
-          {forEach(
-            SCSS_VARIABLES.container,
-            (data: { key: string; value: { fluid: { padding: string } } }) => (
-              <td key={data.key}>{data.value.fluid.padding}</td>
-            ),
-          )}
-        </tr>
-
-        <tr>
-          <th>Gutter width</th>
-          {forEach(
-            SCSS_VARIABLES.grid,
-            (data: { key: string; value: { ['gutter-width']: string } }) => (
-              <td key={data.key}>{data.value['gutter-width']}</td>
-            ),
-          )}
-        </tr>
-
-        <tr>
-          <th>Amount of columns</th>
-          <td colSpan={SCSS_VARIABLES['breakpoint-count']}>{SCSS_VARIABLES['grid-columns']}</td>
-        </tr>
-
-        <tr>
-          <th>Custom gutters</th>
-          <td colSpan={SCSS_VARIABLES['breakpoint-count']}>
-            <LinkTo kind="foundations-layout-gutters" story="docs">
-              yes
-            </LinkTo>
-          </td>
-        </tr>
-        <tr>
-          <th>Nestable</th>
-          <td colSpan={SCSS_VARIABLES['breakpoint-count']}>
-            <a href="#nesting" target="_self">
-              yes
-            </a>
-          </td>
-        </tr>
-        <tr>
-          <th>Column ordering</th>
-          <td colSpan={SCSS_VARIABLES['breakpoint-count']}>
-            <LinkTo kind="foundations-layout-columns" story="docs">
-              yes
-            </LinkTo>
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </div>
+export const GridTable = () => (
+  <SpecTable>
+    <tr>
+      <th>Class prefix</th>
+      {forEach(SCSS_VARIABLES.breakpoint, (data: { key: string; value: { infix: string } }) => (
+        <td key={data.key}>
+          <code>.col-{data.value.infix === 'none' ? '' : `${data.value.infix}-`}</code>
+        </td>
+      ))}
+    </tr>
+
+    <tr>
+      <th>Gutter width</th>
+      {forEach(
+        SCSS_VARIABLES.grid,
+        (data: { key: string; value: { ['gutter-width']: string } }) => (
+          <td key={data.key}>{data.value['gutter-width']}</td>
+        ),
+      )}
+    </tr>
+
+    <tr>
```

Comment:
I don't really understand these 4 lines: 
![image](https://github.com/swisspost/design-system/assets/33580481/d0d41f74-cdac-4f39-b31f-e5c0deb914f8)

Reply:
@alizedebray which 4 lines are we talking about exactly?